SECURITY.md

```diff
@@ -132,8 +135,9 @@ The following components are out of scope for the bounty program
 to, any and all web properties not explicitly listed on this page)
 
 ### Eligibility:
+* Submissions _MUST_ include an exploit proof-of-concept to be considered eligible
 * The participant submitting the bug report shall follow the process outlined within this document
-* Valid exploits can be eligible even if they are not successfully executed on the cluster
+* Valid exploits can be eligible even if they are not successfully executed on a public cluster
```

Comment:
Maybe should restrict it to the ones we care about? Otherwise someone could claim it for their own forks.

Reply:
afaik the security policiy is implied to be against the origin fork. historically, i've successfully told anyone who tried similar (usually referencing old commits) to pound sand with no push back.  agreed that the whole doc could us another rewrite (separately), though